staging/vhost-device-console/README.md

```diff
@@ -0,0 +1,137 @@
+# vhost-device-console - Console emulation backend daemon
+
+## Description
+
+This program is a vhost-user backend that emulates a VirtIO Console device.
+The device's binary takes as parameters a socket path, a socket number which
+is the number of connections, commonly used across all vhost-devices to
+communicate with the vhost-user frontend devices, and the backend type
+"nested" or "network".
+
+The "nested" backend allows input/output to the guest console through the
+current terminal.
+
+The "network" backend creates a local TCP port (specified on vhost-device-console
+arguments) and allows input/output to the guest console via that socket.
+
+This program is tested with QEMU's `vhost-user-device-pci` device.
+Examples' section below.
+
+## Staging Device
+This device will be in `staging` until we complete the following steps:
+- [ ] Enable VIRTIO_CONSOLE_F_SIZE feature
+- [ ] Enable VIRTIO_CONSOLE_F_EMERG_WRITE feature
```

Comment:
What about  `Support` instead of `Enable` ?

Reply:
Agree, done!